docker/production.Dockerfile

```diff
@@ -46,10 +52,13 @@ RUN apt-get update && \
 
 # Install Instantclient Basic Light Oracle and Dependencies
 WORKDIR /opt/oracle
-RUN wget https://download.oracle.com/otn_software/linux/instantclient/instantclient-basiclite-linuxx64.zip && \
-    unzip instantclient-basiclite-linuxx64.zip && rm -f instantclient-basiclite-linuxx64.zip && \
-    cd /opt/oracle/instantclient* && rm -f *jdbc* *occi* *mysql* *mql1* *ipc1* *jar uidrvci genezi adrci && \
-    echo /opt/oracle/instantclient* > /etc/ld.so.conf.d/oracle-instantclient.conf && ldconfig
+
+RUN wget https://download.oracle.com/otn_software/linux/instantclient/instantclient-basiclite-linuxx64.zip 
```

Comment:
Could you combine these multiple run statements to single run?
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers